src/guiapi/include/window_msgbox.hpp

```diff
@@ -9,7 +9,7 @@
 #include "client_response.hpp"
 
 extern const PhaseResponses Responses_NONE;
```

Comment:
Can be please NONE in the same style like the rest.

Reply:
NONE is special case, I prefer it as it is